code/game/gamemodes/objective.dm

```diff
@@ -17,8 +17,25 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 
 /datum/objective/Destroy()
 	GLOB.all_objectives -= src
+	if(owner)
+		owner.objectives -= src
+		owner = null
 	return ..()
 
+/**
+ * Will try to renew the objective to a valid state with another target.
+ * Returns the new found target or null if none were found
+ */
+/datum/objective/proc/on_target_loss()
+	if(owner?.current)
+		to_chat(owner.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+		owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/alarm4.ogg', 100, FALSE, pressure_affected = FALSE)
```

Comment:
`playsound_local` does only play it for the one mob. `playsound` will play it for all in the area.
I believe I took this from how traitors are greeted atm which uses the same system.

Reply:
Ah, I was looking at how cult conversion does it. I tested ingame too but in hindsight I called it on my mob so of course it would still play. I think one of them adds reverb and one doesn't maybe?

Reply:
Indeed it adds reverb. Fox has changed this part of the code a few months ago to also use `SEND_SOUND` so I'll change it to that as well to keep it the same